.ci/azure-pipelines/build_linux.bash

```diff
@@ -20,7 +20,7 @@
 #
 
 if [[ -n "$MUMBLE_BUILD_NUMBER_TOKEN" ]]; then
-	VERSION=$(python "scripts/mumble-version.py" --project)
+	VERSION=$(python "scripts/mumble-version.py" -f version)
```

Comment:
```suggestion
	VERSION=$(python "scripts/mumble-version.py" --format version)
```
I am a big fan of using the long options in scripts in order to make them more verbose for readers.